tests/selenium_tests/agora/Agora.py

```diff
@@ -86,13 +85,7 @@ def basic_authentication(self):
         self.driver.find_element_by_xpath('//md-content//button[text()="login"]').click()
 
         # Response check
-        self.wait.until(EC.presence_of_element_located((By.CLASS_NAME, 'toast-level-success')))
-        login_response_message = self.driver.find_element_by_class_name("toast-level-success").text.split("\n")[0]
-        assert "Login Success" in login_response_message
-        if self.driver.find_element_by_class_name("toast-level-success"):
-            self.driver.find_element_by_xpath('//md-toast//button[text()="close"]').click()
-            # print("[Login] {0:>47} \t\t{1}".format(login_response_message, "Success"))
-            return True
+        self.wait.until(EC.visibility_of_element_located((By.CLASS_NAME, 'toast-level-success')))
```

Comment:
Ok, but this way we will no longer have the message:
```
[Saving form status]                     Form Saved 		Success
```
```suggestion
        try:
            self.wait.until(EC.visibility_of_element_located((By.CLASS_NAME, 'toast-level-success')))
            print("[Saving form status] {0:>30} \t\t{1}".format("Form saved", "Success"))
        except TimeoutException : 
            print("[Saving form status] {0:>38} \t{1}".format("Form was not saved", "Failed"))
            print(TimeoutException.message)
```

Reply:
You are right I added it to the file